dokuwiki/templates/NOTES.txt

```diff
@@ -0,0 +1,21 @@
+
+Thanks for installing Jupyter notebook with Dataporten integration.
```

Comment:
Somewhat nitpicky, but we might want to change this to dokuwiki (jupyter is also mentioned in values.yaml)